src/pixelator/report/__init__.py

```diff
@@ -551,7 +552,10 @@ def _collect_metrics(adata: AnnData) -> Dict[str, float]:
                 adata.obs["tau_type"]
             )
             metrics["reads_of_aggregates"] = adata[aggregates_mask].obs["reads"].sum()
-            metrics["umis_of_aggregates"] = adata[aggregates_mask].obs["umi"].sum()
+            metrics["edges_of_aggregates"] = adata[aggregates_mask].obs["edges"].sum()
```

Comment:
Very small thing, but shouldn't in be `edge_in_aggregates`?

Reply:
Good suggestion. I renamed several metrics and realized there was one left unused so removed that one too.